### PR TITLE
MenuItem and relevant repo, service, controller implemented

### PR DESCRIPTION
**Pull request by javasia**
MenuItem and relevant repo, service, controller implemented but there're no tests yet

### CODE REVIEW

**Comment by javasia** [comment]
[3SP - Add MenuItem entity, repository, service and controller](https://trello.com/c/u5xveZdc/126-3sp-add-menuitem-entity-repository-service-and-controller)

File: src/main/java/com/danit/finalproject/application/entity/MenuItem.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+@Table(name = "menu_items")
+@Data
+@NoArgsConstructor
+public class MenuItem extends BaseEntity {
+
+  @Column(name = "name")
+  private String name;
+
+  @Column(name = "display_name")
+  private boolean displayName;
```
Comment (skosinskiy) [inline src/main/java/com/danit/finalproject/application/entity/MenuItem.java]: I guess here we should have String field

Comment (javasia) [reply src/main/java/com/danit/finalproject/application/entity/MenuItem.java]: Then should I put into testing routine?

Comment (skosinskiy) [reply src/main/java/com/danit/finalproject/application/entity/MenuItem.java]: You can leave test the same as now, just change data in sql and

Comment (javasia) [reply src/main/java/com/danit/finalproject/application/entity/MenuItem.java]: changes implemented as per your review

**Pull request merged**